### DISCUSSION TITLE
Executing the scala code for the first time reports an error

### DISCUSSION THREAD

**Discussion by winfys** [apache/kyuubi#5174] [Q&A] [answered]
kyuubi version:1.6.1
spark version:3.3.1

The first execution reports an error, and it is normal to execute again
![image](https://github.com/apache/kyuubi/assets/13195083/9c91c293-834e-4f8e-912c-3159d0abd707)

**Comment by winfys** [comment, marked as answer]
It's a problem with the compiled jdk version

**Reply by pan3793** [reply]
Thanks for your question and clarification, it would be nice if you can provide more detail about "a problem with the compiled JDK version", in case someone wants to explore it in the future.

**Reply by winfys** [reply]
sorry,I have done some development on kyuubi spark SQL engine, which is caused by the jdom package
![image](https://github.com/apache/kyuubi/assets/13195083/94fc56a5-e7ae-4baf-a8d1-157e243b3bcf)
